pyanaconda/ui/gui/spokes/network.py

```diff
@@ -172,8 +173,8 @@ class CellRendererSecurity(Gtk.CellRendererPixbuf):
     __gproperties__ = {
         "security": (GObject.TYPE_UINT,
                    "Security", "Security",
```

Comment:
I'd move this one 2 spaces right as well.

Reply:
Fixed.